### DISCUSSION TITLE
FEM and Strain based constraints in XPBD

### DISCUSSION THREAD

**Discussion by mjcg91** [InteractiveComputerGraphics/PositionBasedDynamics#122] [Q&A] [answered]
Hi,

I would like to know if the FEM and strain based constraints could be updated simply by using XPBD style lambda update.I have read contracticting opinion whether C should be computed differently in PBD and XPBD or not.

Thank you

**Comment by janbender** [comment, marked as answer]
Yes, in principle this is possible. Just be aware that in this case you don't have a single compliance value but a compliance matrix.

**Reply by swq0553** [reply]
So，have any plans add these constraints`s xpbd verson?

**Reply by janbender** [reply]
We implemented the FEM constraints for XPBD a few months ago. So check out the newest version.
